utils/reputation/client.go

```diff
@@ -112,7 +112,13 @@ func (c *HTTPClient) do(req *http.Request, v interface{}) (*http.Response, error
 		}
 		return resp, nil
 	}
-	return resp, fmt.Errorf("Request error: %d", resp.StatusCode)
+	return resp, err
+}
+
+// ResponseErrorBundle structure preseves important parts of the error
```

Comment:
this doesn't appear to be used and is confusingly named due to the other `ResponseErrorBundle`